### DISCUSSION TITLE
mjpeg camera what are steps to let it works

### DISCUSSION THREAD

**Discussion by GMagician** [blakeblackshear/frigate#14989] [Ask A Question] [answered]
Hello,

I have a mjpeg camera and I can see it through vlc using proper http uri.
To let it work with frigate I added such uri in go2rtc (mjpeg_cam: "ffmpeg:http://xxxx#video=h264#hardware")
I also added in camera specific config: "input_args: preset-http-jpeg-generic" but it doesn't work.

Anything else I missed?

**Comment by NickM-27** [comment]
because you are using go2rtc, your input args should be the standard `preset-rtsp-restream`

**Reply by GMagician** [reply]
Thanks for your support, if I recall correctly first attempt was without it, but result was the same. Anything else to care of ?

**Reply by NickM-27** [reply]
need to see config and logs

**Reply by GMagician** [reply, marked as answer]
Thanks again, while preparing data to attach I discovered that the problem was my straem definition. Now It works.
